keyboards/sol/keymaps/danielhklein/keymap.c

```diff
@@ -0,0 +1,364 @@
+#include QMK_KEYBOARD_H
+#include "bootloader.h"
```

Comment:
This line isn't needed; the `QMK_KEYBOARD_H` include on Line 1 includes this file automatically.

```suggestion
```